dataservice/api/participant/models.py

```diff
@@ -14,3 +14,5 @@ class Participant(db.Model, Base):
     """
     __tablename__ = "participant"
     external_id = db.Column(db.String(32))
+    sample = db.relationship('Sample', backref='participant', uselist=False,
```

Comment:
I think you want to use `uselist=True` here right? Since Participant --> Sample is a one to many relationship

Reply:
And then if that is true, change the variable name `sample` to `samples`

Reply:
yep one to many relationship between Participant and Sample.